hack/manifest-gen/assets/charts/eck/templates/configmap.yaml

```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.operator.name }}
```

Comment:
```
> k get cm -n elastic-system 
NAME                    DATA   AGE
elastic-operator        1      5m
elastic-operator-uuid   1      19s
```

Should we call the ConfigMap `elastic-operator-configuration` to make its purpose more explicit ?

Reply:
I have seen quite a fair amount of applications simply using the name of the application and I think that convention makes sense. It is a config map after all so the purpose is obvious without the additional `-configuration` suffix. If there are strong opinions against that, I'd be happy to reconsider.

Reply:
> It is a config map after all so the purpose is obvious

I'm not sure I would agree with that, we already have 2 ConfigMaps which are not really related to the configuration and we might have a third one for the election mechanism:

```
k get cm -n elastic-system
NAME                      DATA   AGE
elastic-licensing         4      18h
elastic-operator-leader   0      18h
elastic-operator-uuid     1      18h
```

I would be curious what other people's thoughts are, but I don't want to block this PR.

Reply:
I see your point. I am probably building up the convention in my mind because I have been conditioned by it but that probably doesn't apply to everybody. I'll merge this for now and if it turns out to be confusing, we still have time to change it before the next release goes out. 

Simply because I have it in hand, Istio config maps look like the following. 

```
istio                                  2      63d
istio-ca-root-cert                     1      63d
istio-leader                           0      63d
istio-namespace-controller-election    0      63d
istio-security                         1      63d
istio-sidecar-injector                 2      63d
istio-validation-controller-election   0      63d
kiali                                  1      58d
prometheus                             1      63d
```

I am not suggesting that this is a canonical example. However, I think it's quite obvious what each config map does. I reckon that ours would look similar.